lib/packages/docutils/highlite.nim

```diff
@@ -45,15 +57,15 @@ from algorithm import binarySearch
 type
   SourceLanguage* = enum
     langNone, langNim, langCpp, langCsharp, langC, langJava,
-    langYaml, langPython
+    langYaml, langPython, langCmd
   TokenClass* = enum
     gtEof, gtNone, gtWhitespace, gtDecNumber, gtBinNumber, gtHexNumber,
     gtOctNumber, gtFloatNumber, gtIdentifier, gtKeyword, gtStringLit,
     gtLongStringLit, gtCharLit, gtEscapeSequence, # escape sequence like \xff
     gtOperator, gtPunctuation, gtComment, gtLongComment, gtRegularExpression,
     gtTagStart, gtTagEnd, gtKey, gtValue, gtRawData, gtAssembler,
     gtPreprocessor, gtDirective, gtCommand, gtRule, gtHyperlink, gtLabel,
-    gtReference, gtOther
+    gtReference, gtProgram, gtOption, gtOther
```

Comment:
how about using same approach as https://github.com/nim-lang/Nim/pull/15775 for these to simplify code:
```
  TokenClass* = enum
    gtEof = "Eof"
    ...
```
(the raw "gtEof" can always be had with `enumutils.symbolName` in the rare case it's needed)

Reply:
@timotheecour yes, that would be a better style.